frontend/app/work_packages/controllers/dialogs/columns.js

```diff
@@ -35,13 +35,16 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
   vm.selectedColumns = [];
   vm.oldSelectedColumns = [];
   vm.availableColumns = [];
+  vm.unusedColumns = [];
```

Comment:
Available columns are all columns you could ever select. Unused columns are those that are not currently selected.

So basically: `unusedColumns = availableColumns - selectedColumns`